pyproject.toml

```diff
@@ -30,7 +31,8 @@ APScheduler = "*"
 redis = "*"
 matplotlib = "*"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
```

Comment:
updated to newer style of poetry

ref: https://python-poetry.org/docs/managing-dependencies/#adding-a-dependency-to-a-group